webscalesqlclient/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ cmake_minimum_required(VERSION 2.8.0)
 include(ExternalProject)
 ExternalProject_Add(
   webscalesqlclient
+  URL file://${CMAKE_CURRENT_SOURCE_DIR}/webscalesql-5.6
```

Comment:
Hey @jwatzman as a KDE developer my habit is

```
mkdir build
cd build
cmake .. -DCMAKE_INSTALL_PREFIX=/usr balabala
```

![cmake-screenshot1](https://cloud.githubusercontent.com/assets/851100/7875170/04d421b4-05ec-11e5-816c-96966305f622.png)

such as what a ArchLinux packager did https://projects.archlinux.org/svntogit/packages.git/tree/trunk/PKGBUILD?h=packages/kdelibs#n27

Regards,
Leslie Zhai